src/styles/index.scss

```diff
@@ -1,3 +1,79 @@
+$hand-transform-origin: bottom center;
+$hand-bottom: 50%;
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  height: 80vmin;
+  width: 80vmin;
+  border: dotted #000 1vmin;
+  border-radius: 50%;
+
+  &__hand-minute {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    position: absolute;
+    bottom: $hand-bottom;
+    animation: rotate-minute 3600s infinite linear;
+    transform-origin: $hand-transform-origin;
```

Comment:
The 'transform-origin' property is a shorthand for the 'transform-origin-x' and 'transform-origin-y' properties. It is recommended to use the shorthand form when setting both values to the same. In this case, 'bottom center' can be simplified to 'center bottom' to follow the conventional order (x y).